src/main/java/friday/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package friday;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks in the Friday application.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList = new ArrayList<>();
+
+    public void add(Task task) {
+        taskList.add(task);
+    }
+
+    /**
+     * Mark a specific task in the list as done.
+     * @param taskNumber The index of the task to be marked as done.
```

Comment:
Should the Javadoc start with `Marks` instead? Also, should there be an extra line between the first line of text and the second line? I have noticed this issue in some of the other chunks of code.